packages/dds/tree/src/feature-libraries/sequence-field/format.ts

```diff
@@ -86,18 +125,25 @@ export interface Insert<TNodeChange = NodeChangeType>
     content: ProtoNode[];
 }
 
-export interface MoveIn extends HasMoveId, HasPlaceFields, HasRevisionTag {
+export interface MoveIn extends HasMoveId, HasPlaceFields, HasRevisionTag, Mutable {
     type: "MoveIn";
     /**
      * The actual number of nodes being moved-in. This count excludes nodes that were concurrently deleted.
      */
     count: NodeCount;
+    /**
+     * When true, the corresponding MoveOut has been muted.
+     * This is independent of whether this mark is muted.
```

Comment:
I agree. Maybe we should reserve "muted" for the operation (meaning `isMuted(moveIn) === moveIn.mutedBy !== undefined || moveIn.isSrcMuted` though the names of those fields mould likely change) and have something else for the mark itself. Any ideas?

Reply:
I think we should have a term to mean that the mark has no effect when applied to the field, such as muted or disabled. I think something like conflicted (although that might also have poor connotations in context of transactions) seems closer to what this PR means by muted (since muted marks are always muted by a specific change they rebased over). It makes sense to me to say that one endpoint of a move has a conflict, so the entire move is muted.

I'm curious how much code in this PR uses the Muteable interface without knowing the mark type. I suspect that generally the rebasing module needs to understand the reason for a mark being muted/conflicted, and maybe we can replace Muteable with different types of "conflicts." One obvious category of conflicts is when a mark wants to manipulate a node which has been concurrently deleted. There are probably multiple reasonable ways to factor the types of conflicts return and revive can have, but we could have a conflict type for trying to undo a change which has already been undone (for non-intention inverses), and another type for trying to revive content which have already been revived or fill a cell which has already been filled (for intention inverses).

Reply:
I do like "conflicted". I think it really does represent what's going on. In the context of transactions I actually think makes sense because we want to support aborting the transaction if such a conflict is encountered (like legacy SharedTree does) as well as applying all the non-conflicted marks.

Are you thinking of having an enum for the types of conflicts, and having marks carry an optional field for it? I'm not sure how much the code would benefit from that since it tends to determine the mark type first, and the kind of conflict that a given mark can have is deducable from its type.
When we start supproting atomicity for transactions I do want to make sure we expose information about the conflict(s) that brought the transaction down.

Reply:
I'm not sure what the implementation should be. I'm mostly objecting to the conceptual model that the name Muteable brings to mind for me.

Reply:
Cool. I'll change the terminilogy to "conflict".